src/components/custom-field-input-text/custom-field-input-text.css

```diff
@@ -0,0 +1,71 @@
+@import '../../styles/colors.css';
+@import '../../styles/spacing.css';
+@import '../../styles/typography.css';
+
+.custom-field-input-text {
+  font-family: var(--mavenlink-type-font-family);
+}
+
+.heading-container {
+  color: var(--palette-grey-dark);
+  font-size: var(--mavenlink-type-body-font-size);
+}
+
+.error .heading-container {
+  color: var(--palette-caution-base);
+}
+
+.help {
+  color: var(--palette-grey-dark);
+  font-size: var(--mavenlink-type-base-size);
+}
+
+.error .help {
+  color: var(--palette-caution-base);
+}
+
+.input {
+  border: 1px solid var(--palette-grey-light);
+  border-radius: var(--spacing-x-small);
+  color: var(--palette-grey-x-dark);
+  font-size: var(--mavenlink-type-base-size);
+  padding: var(--spacing-medium);
+}
+
+.disabled .input {
+  background-color: var(--palette-grey-light);
+}
+
+.error .input {
+  border-color: var(--palette-caution-base);
+  padding-right: var(--spacing-x-large);
+}
+
+.input-container {
+  height: 34px;
+  margin: var(--spacing-small) 0;
+}
+
+.input-container input::placeholder {
+  color: var(--palette-grey-dark);
+}
+
+.input-icon-container {
+  background-color: var(--white);
+  display: inline-block;
+  height: 20px;
```

Comment:
This isn't really a blocker and more of nit-picking: 

Usually, we want to avoid setting a height + width on an absolutely position element in favor of letting the browser do its own thing.

In this case, we are not correctly calculating the boundaries of the actual icon with these rules:

<img width="328" alt="Screen Shot 2019-12-13 at 2 58 07 PM" src="https://user-images.githubusercontent.com/841084/70837769-08fad400-1dba-11ea-9ee5-e21f5ea79a83.png">

But if we remove height + width then the document flows naturally:

<img width="358" alt="Screen Shot 2019-12-13 at 2 58 19 PM" src="https://user-images.githubusercontent.com/841084/70837778-1617c300-1dba-11ea-9171-538d3477a1f3.png">

This is not blocking since it doesn't affect anything but it's a habit that is not optimal. For sake of maintenance, default browser behavior is usually preferred -- from my own experience.